R/generate.R

```diff
@@ -0,0 +1,55 @@
+int_types <- c("integer", "smallint")
```

Comment:
These are likely db specific?  Not something we need to generalise here right now, but something that would be an issue if we were to make this more general later?

Reply:
Yeah at the moment this is very much tied to Postgres. This is also a relatively limited set of Postgres types, just those we use in Montagu. Will add something to that effect in the README but would be nice (and easy enough) to generalise later on.